obfx_modules/elementor-extra-widgets/widgets/elementor/pricing-table.php

```diff
@@ -1039,23 +1039,25 @@ protected function render() {
 			$output .= '<div class="obfx-title-wrapper">';
 			if ( ! empty( $settings['title'] ) ) {
 				// Start of title tag.
-				$output .= '<' . esc_html( $settings['title_tag'] ) . ' ' . $this->get_render_attribute_string( 'title' ) . '>';
+				$title_tag = in_array( $settings['title_tag'], array( 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'p' ), true ) ? $settings['title_tag'] : 'h1';
```

Comment:
one small suggestion is to create a small function to reuse this checking of tag value (since it is used for both title and subtitle tags). Like: `$title_tag = $this->sanitize_tag(  $settings['title_tag'] )`

Reply:
@Soare-Robert-Daniel Thank you for your feedback! I've implemented your suggestion and pushed the updated code.